tox.ini

```diff
@@ -1,10 +1,10 @@
 [tox]
-envlist = py26, py27, py33, py34, py35
+envlist = py27,py33,py34,py35
 
 [testenv]
 deps =
-    mock
     pyzmq
-    unittest2
+    pytest
+    py27: mock
 commands =
-    unit2 discover []
+    pytest {posargs}
```

Comment:
pytest is really awesome.
Most important feature IMO, you can write concise tests that are easy to read and so to maintain.